src/lightning/pytorch/strategies/launchers/xla.py

```diff
@@ -68,16 +68,22 @@ def launch(self, function: Callable, *args: Any, trainer: Optional["pl.Trainer"]
                 a selected set of attributes get restored in the main process after processes join.
             **kwargs: Optional keyword arguments to be passed to the given function.
         """
-        context = mp.get_context(self._start_method)
-        return_queue = context.SimpleQueue()
+        return_queue = mp.Manager().Queue()
         import torch_xla.distributed.xla_multiprocessing as xmp
+        from torch_xla.experimental import pjrt
+
+        spawn_kwargs = {}
+        if pjrt.using_pjrt() and (nprocs := self._strategy.num_processes) == 1:
```

Comment:
For non PJRT case, the `nprocs` arg would still be needed.

Reply:
Yes, I noticed. This is why I added the `using_pjrt` check

Reply:
I'm a little confused. In the XRT case `pjrt.using_pjrt()` is `False`, wouldn't the `spawn_kwargs` be empty?

Reply:
Right. Let me make this explicit. Thanks